bin/extract.js

```diff
@@ -12,7 +12,8 @@ var options = cli.parse({
 	dir: ['d', 'Directory path you would like to extract strings from', 'dir', process.env.PWD],
 	output: ['o', 'Directory path you would like to save extracted strings', 'dir', process.env.PWD],
 	format: ['f', 'Output format', ['json', 'pot'], 'json'],
-	merge: ['m', 'Merge options, true to merge, false to replace, clean to remove unused keys while merging', [true, false, 'clean'], true]
+	replace: ['r', 'Replace the content of the file if it exists (merging by default)'],
```

Comment:
My next PR is the ability to provide a list of languages so that it generates a file for each (or update each existing file) instead of just generating "template.xxx" :D